### DISCUSSION TITLE
Distinguish between Issue and Discussion in notification emails

### DISCUSSION THREAD

**Discussion by rappen** [community/community#2900] [Discussions] [answered]
As a repo owner I want to encourage use of Discussions to keep Issues less cluttered.
It would be nice to be able to distinguish between notification emails for Discussions and Issues, so I can keep directing issue notifications to a subfolder in my email but be more alert to responding to discussions.

**Comment by becca** [comment, marked as answer]
Conversation around this is happening in https://github.com/github/feedback/discussions/59

For now we do support filtering Discussions separately from all other notification types if you use web notifications on github.com
